docs/src/main/tut/client.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: default
+title: HTTP Client
+---
+
+## Your first client
+
+How do we know the server is running?  Let's create a client with
+http4s to try our service.
```

Comment:
Okay, I see we define the service further down.  That leaves this paragraph kind of adrift.